package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "scripts": {
     "test": "yarn packages --parallel run test",
-    "packages": "yarn workspaces foreach --recursive --from '{@relay-protocol/addresses,@relay-protocol/eslint-config,@relay-protocol/helpers,@relay-protocol/networks,@relay-protocol/tsconfig,@relay-protocol/types,@relay-protocol/abis,@relay-protocol/client}'",
+    "packages": "yarn workspaces foreach --recursive --from '{@relay-protocol/addresses,@relay-protocol/eslint-config,@relay-protocol/helpers,@relay-protocol/networks,@relay-protocol/tsconfig,@relay-protocol/types,@relay-protocol/abis,@relay-protocol/client,@relay-protocol/arb-claimer}'",
```

Comment:
Since this could do more than just `claim` let's be more generic with `helper-arb`